cms/templates/admin/cms/page/change_form.html

```diff
@@ -12,9 +12,13 @@
 
 {% block extrahead %}
 {{ block.super }}
+<!-- load nonconflict pre and post files on line 17 and 21 to fix conflict if there is another jQuery loaded -->
```

Comment:
don't use html comments, use django comments instead.
also, line numbers tend to change so no need to specify them